demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -257,3 +260,20 @@ def test_is_all_params_not_hidden(self, source, answer):
         structure = StructureValidator(source)
         validator = IntegrationValidator(structure)
         assert validator.is_all_params_not_hidden() is answer
+
+    INPUTS_RELEASE_NOTES_VALIDATION = [
+        (VALID_INTEGRATION_TEST_PATH, INTEGRATION_TARGET),
+        (VALID_SCRIPT_PATH, SCRIPT_TARGET),
+        (VALID_DASHBOARD_PATH, DASHBOARD_TARGET),
+        (VALID_INCIDENT_FIELD_PATH, INCIDENT_FIELD_TARGET),
+        (VALID_TEST_PLAYBOOK_PATH, PLAYBOOK_TARGET)
+    ]
+
+    @pytest.mark.parametrize('source, target', INPUTS_RELEASE_NOTES_VALIDATION)
+    def test_is_file_valid_rn(self, source, target):
+        # type: (str, str) -> None
+        try:
+            copyfile(source, target)
+            assert FilesValidator(validate_conf_json=False).is_valid_structure()
```

Comment:
this just validates schema, and doesnt actually get to the release notes validator 
what we need to test here is `validate_added_files`

Reply:
it does. it tests the structure and then goes to valid file and tests everything. If there is a valid new file and it has release notes it will fail

Reply:
unless im missing something, for every test case it runs `validate_all_files`, which does not validate release notes, but only scheme

Reply:
We don't need the releas notes validator cause I'm not interested if the release notes file is valid (which is not cause it doesn't exists).
The `is_valid_structure` initiates the `self.validate_added_files({self.file_path}, file_type=find_type(self.file_path))` that check all the added files including the rn validation

Reply:
`is_valid_structure` calls `validate_added_files` only if `file_path` is provided, but it is not in your case, so it will always continue to the `else` clause and run `validate_all_files`